music/demos/music_vae.ts

```diff
@@ -458,11 +458,120 @@ async function runMultitrackChords() {
+  drumsButton.textContent = 'Generate Drums';
+  drumsButton.addEventListener('click', () => {
+    runDrums();
+    drumsButton.disabled = true;
+  });
+  const drumsDiv = document.getElementById('generate-drums');
+  drumsDiv.appendChild(drumsButton);
+}
+
+async function generateDrumsNadeButton() {
+  const drumsNadeButton = document.createElement('button');
+  drumsNadeButton.textContent = 'Generate Drums NADE';
+  drumsNadeButton.addEventListener('click', () => {
+    runDrumsNade();
+    drumsNadeButton.disabled = true;
+  });
+  const drumsNadeDiv = document.getElementById('generate-drums-nade');
+  drumsNadeDiv.appendChild(drumsNadeButton);
+}
+
+async function generateMelButton() {
+  const melButton = document.createElement('button');
+  melButton.textContent = 'Generate Melody';
+  melButton.addEventListener('click', () => {
+    runMel();
+    melButton.disabled = true;
+  });
+  const melDiv = document.getElementById('generate-melody');
+  melDiv.appendChild(melButton);
+}
+
+async function generateMelChordsButton() {
+  const melChordsButton = document.createElement('button');
+  melChordsButton.textContent = 'Generate Chord-Conditioned Melody';
+  melChordsButton.addEventListener('click', () => {
+    runMelChords();
+    melChordsButton.disabled = true;
+  });
+  const melChordsDiv = document.getElementById('generate-melody-chord');
+  melChordsDiv.appendChild(melChordsButton);
+}
+
+async function generateMel16Button() {
+  const mel16Button = document.createElement('button');
+  mel16Button.textContent = 'Generate Melody 16-bar';
+  mel16Button.addEventListener('click', () => {
+    runMel16();
+    mel16Button.disabled = true;
+  });
+  const mel16Div = document.getElementById('generate-melody-16');
+  mel16Div.appendChild(mel16Button);
+}
+
+async function generateTrioButton() {
+  const trioButton = document.createElement('button');
+  trioButton.textContent = 'Generate Trio';
+  trioButton.addEventListener('click', () => {
+    runTrio();
+    trioButton.disabled = true;
+  });
+  const trioDiv = document.getElementById('generate-trio');
+  trioDiv.appendChild(trioButton);
+}
+
+async function generateMultitrackButton() {
+  const multitrackButton = document.createElement('button');
+  multitrackButton.textContent = 'Generate Multitrack';
+  multitrackButton.addEventListener('click', () => {
+    runMultitrack();
+    multitrackButton.disabled = true;
+  });
+  const multitrackDiv = document.getElementById('generate-multitrack');
+  multitrackDiv.appendChild(multitrackButton);
+}
+
+async function generateMultitrackChordsButton() {
+  const multitrackChordsButton = document.createElement('button');
+  multitrackChordsButton.textContent = 'Generate Chord-Conditioned Multitrack';
+  multitrackChordsButton.addEventListener('click', () => {
+    runMultitrackChords();
+    multitrackChordsButton.disabled = true;
+  });
+  const multitrackChordsiv = document.getElementById(
```

Comment:
multitrackChordsDiv (though it doesn't really matter because #87 is going to move this to another file anyway)